app/packs/src/components/ChemicalTab.js

```diff
@@ -788,23 +789,24 @@ export default class ChemicalTab extends React.Component {
     }
     const savedSds = chemical._chemical_data[0].safetySheetPath;
     const sdsStatus = safetySheets.length ? safetySheets : savedSds;
-    const mappedSafetySheets = sdsStatus.map((document, index) => {
+    const mappedSafetySheets = sdsStatus?.map((document, index) => {
       const key = (document.alfa_product_number || document.merck_product_number) || index;
       return (
         <div className="safety-sheets-form" key={key}>
-          {document !== 'Could not find safety data sheet from Thermofisher' && document !== 'Could not find safety data sheet from Merck' ? (
-            <ListGroupItem key={`${key}-file`}>
-              {this.renderChildElements(document, index)}
-            </ListGroupItem>
-          ) : (
-            <ListGroupItem key={`${key}-no-document`}>
-              <div>
-                <p className="safety-sheets-paragraph">
-                  {document}
-                </p>
-              </div>
-            </ListGroupItem>
-          )}
+          {document !== 'Could not find safety data sheet from Thermofisher'
```

Comment:
May be wrapping both checks for "document" with a scope?

Reply:
done